apps/api/src/routes/middleware/errorHandler.ts

```diff
@@ -11,9 +11,9 @@ export function errorHandler(error: Error, request: Request, response: Response,
     }
 
     if (error instanceof QueryFailedError) {
-        const error = new BadRequestError({ message: ServerErrorMessage.invalidData()});
+        const error = new BadRequestError({ message: ServerErrorMessage.invalidData() });
         return response.status(error.httpStatus).json(error.toResponse());
     }
 
-    return response.status(StatusCodes.INTERNAL_SERVER_ERROR).json('Something went wrong.. Try again alter');
-}
\ No newline at end of file
+    return response.status(StatusCodes.INTERNAL_SERVER_ERROR).json('Something went wrong.. Try again later.' + error);
+}
```

Comment:
Is this safe? Was just thinking its not a good idea to expose the error that cause the 502

Reply:
Woops, good catch. That is left over from debugging. However, maybe we should look into separate logging for the dev environment?